### PR TITLE
setup the restaurant project

### PR DESCRIPTION
**Pull request by Oghenebrume50**

### CODE REVIEW

**Comment by Oghenebrume50** [comment]
Hi reviewer, I focused on the javascript logic since this is not a portfolio-worthy project, I just decided to get the main logic underway. I hope this helps :smile:

File: eslint.config [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
```
[the remark this answers is not in the document]
Comment (Oghenebrume50) [reply eslint.config]: Hi this is the configuration specified by microverse see link here https://github.com/microverseinc/linters-config/blob/master/javascript/eslint.config

**Comment by Oghenebrume50** [comment]
Thank you for the insight

**Comment by SunnyAsar** [comment]
### Hey @Oghenebrume50  😃,
I'm super happy to see how you cleaned you code based on the past review, your Index is super clean now and importing the needed modules. Props! 👊 
#### Observations  
- let's chose to use more of Es6 syntax where possible e.g arrow functions, this is not a deal breaker for me anyway. 
- as pointed out in the previous review you UI could be better, at times like this, use bootstrap for fast styling.

### Finally
 # You may Pass ! 👍  Cheers!

**Pull request merged**